### ISSUE TITLE
Code Update: Switching to androidx lib

### ISSUE THREAD

**Issue by yingding** [googlecodelabs/navigation-action#2]
This navigation-action codelab is outdated. For my own learning purpose, i have upgraded and extended the base and final code to androidx and android studio 2.4.1

The upgraded example working with androidx lib can be found under https://github.com/yingding/android-examples/tree/master/codeLabWearOSNavigationExample

If it is wished, i can also make a pull request for the code repo, but the codelab team needs to update the codelab text.

**Comment by harri35** [comment]
I agree, the code is outdated and needs to be updated to AndroidX and to use the new `androidx.wear.widget.drawer.WearableActionDrawerView` and `androidx.wear.widget.drawer.WearableNavigationDrawerView` classes. Otherwise, a good tutorial, helped me to get a better understanding of the navigation patterns on WearOS
